File: TsogosunProfileAdmin/MSMDMPatron/Model/Dtos/PatronSearchDto.cs
using System;

namespace tsogosun.com.MSMDMPatron.Model.Dtos
{
    public class PatronSearchDto
    {
        public long? Tsogosunid { get; set; }
        public long PatronID { get; set; }
        public int? Property_Registered { get; set; }
        public string SiteName { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string FullName { get; set; }
        public string FIC_IDNumber { get; set; }
        public string CellNumber { get; set; }
        public int CustomerRank { get; set; }
        public string CustomerTier { get; set; }
        public string ProfileStatus { get; set; }
        public DateTime? LastPlayDate { get; set; }
        public string IdentityNumber
        {
            get
            {
                return FIC_IDNumber;
            }
        }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/mdm-patron/patronSearchDto.ts
export interface PatronSearchDto {
    tsogosunid?: number | undefined;
    patronID: number;
    property_Registered?: number | undefined;
    siteName?: string | undefined;
    firstName?: string | undefined;
    middleName?: string | undefined;
    lastName?: string | undefined;
    fullName?: string | undefined;
    fIC_IDNumber?: string | undefined;
    cellNumber?: string | undefined;
    customerRank: number;
    customerTier?: string | undefined;
    profileStatus?: string | undefined;
    lastPlayDate?: Date | undefined;
    identityNumber?: string | undefined;
    
}